text_collectors/virt.sh

```diff
@@ -49,6 +49,19 @@ then
     echo "virt_platform{type=\"openstack\"} 1" >>"${v}"
   fi
 
+  if dmidecode -s system-manufacturer | grep "VMware, Inc."; then
+    count=$(( count + 1 ))
+    case $(dmidecode -t 0 | grep Address | awk '{ print $2 }') in
+    	"0xEA580" ) esxi_version="6.5" ;;
+        "0xEA520" ) esxi_version="6.7" ;;
+        "0xEA490" ) esxi_version="6.7U3" ;;
+        "0xEA480" ) esxi_version="7" ;;
+                * ) echo "unknown"
```

Comment:
Instead of echoing here, it might be better to set `esxi_version="unknown"`, otherwise the esxi_version in the virt_platform metric will be empty.

Reply:
To add to what @dgrisonnet said, AFAICT, esxi_version being unset will cause the script to exit with error